js/markupBlog.js

```diff
@@ -0,0 +1,328 @@
+    authorIconBlog.className = "author_icon";
+
+    img.setAttribute("src", elem.image.src);
+    img.setAttribute("alt", "blog img");
+    img.setAttribute("width", 560);
+    imgIconImg.setAttribute("src", elem.image.icon);
+    imgIconImg.setAttribute("alt", "icon");
+
+    blogTitle.textContent = elem.title;
+    blogText.textContent = elem.text;
+    readMore.setAttribute("href", "");
+    iconAuthorBlog.setAttribute("src", elem.icon);
+    iconAuthorBlog.setAttribute("alt", "icon");
+    audio.setAttribute("src", elem.audio);
+    audio.setAttribute("controls", "controls");
+
+    readMore.textContent = "Read more";
+
+    authorIconBlog.append(iconAuthorBlog);
+
+    infoBlog.append(author);
+    infoBlog.append(blogTitle);
+    elem.audio ? infoBlog.append(audio) : null;
+    infoBlog.append(blogText);
+    infoBlog.append(readMore);
+    infoBlog.append(authorIconBlog);
+    elem.image.icon ? blogIconImg.append(imgIconImg) : null;
+    blog.append(blogIconImg);
+    blog.append(img);
+    blog.append(infoBlog);
+    containerSection.append(blog);
+  });
+
+  data.blog.content.commentBlog.forEach((elem) => {
+    const blogComment = document.createElement("div");
+    const blogCommentAuthor = document.createElement("div");
+    const commentIcon = document.createElement("img");
+    const authorCommentIcon = document.createElement("div");
+    const authorName = document.createElement("p");
+    const commentTitle = document.createElement("h2");
+    const commentText = document.createElement("p");
+    const readMore = document.createElement("a");
+    const author = markupAuthor(elem);
+
+    blogComment.className = "blog_comment";
+    blogCommentAuthor.className = "blog_comment_author";
+    authorName.className = "author_name";
+    commentTitle.className = "author_blog_title";
+    commentText.className = "author_blog_text_comment";
+    readMore.className = "button-light author_btn";
+    authorCommentIcon.className = "author_icon";
+
+    authorName.textContent = elem.author.name;
+    commentTitle.textContent = elem.title;
+    commentText.textContent = elem.text;
+    readMore.textContent = "Read more";
+    commentIcon.setAttribute("src", elem.icon);
+    commentIcon.setAttribute("alt", "icon");
+
+    authorCommentIcon.append(commentIcon);
+    blogComment.append(author);
+    blogComment.append(commentTitle);
+    blogComment.append(commentText);
+    blogComment.append(readMore);
+    blogComment.append(authorCommentIcon);
+    containerSection.append(blogComment);
+  });
+
+  container.append(markupTitle(data.blog.title.primary, "title_border-blog"));
+  readMoreContainer.append(readMoreBtn);
+  search.append(input);
+  searchIcon.append(imgIconSearch);
+  search.append(searchIcon);
+  container.append(search);
+  section.append(containerSection);
+  main.append(container);
+  main.append(section);
+  main.append(readMoreContainer);
+
+  return main;
+};
+
+const markupFooter = () => {
```

Comment:
this function has a duplicate in all markup files too, please move it to separate file.